.gitignore

```diff
@@ -23,3 +23,6 @@ robot/Crisis-Management/results/
 
 # Misc
 .DS_Store
+IlluminatedCloud/TractionThrive__dev/OfflineSymbolTable.zip
+/node_modules/*
```

Comment:
was this intended to be in this PR?

Reply:
it's just to ignore those IJ items in git